spec/requests/schemes_controller_spec.rb

```diff
@@ -2224,6 +2224,31 @@
         end
       end
 
+      context "when there are no stock owners" do
+        before do
+          get "/schemes/#{scheme.id}/edit-name"
+        end
+
+        it "does not include the owning organisation question" do
```

Comment:
Does this want a context of "and there are no recently absorbed organisations"? Purely for reading it purposes.

Reply:
Actually, does there want to be a test where there are older absorbed organisations?